workflow_imaging/paths.py

```diff
@@ -38,8 +38,23 @@ def get_scan_box_files(scan_key):
     if sbx_filepaths:
         return sbx_filepaths
     else:
-        raise FileNotFoundError(f'No .sbx file found in {sess_dir}')
+        raise FileNotFoundError(f'No ScanBox file (.sbx) found in {sess_dir}')
 
+def get_miniscope_daq_file(scan_key):
+    # Folder structure: root / subject / session / .json
+    data_dir = get_imaging_root_data_dir()
+
+    from .pipeline import Session
+    sess_dir = data_dir / (Session.Directory & scan_key).fetch1('session_dir')
+
+    if not sess_dir.exists():
+        raise FileNotFoundError(f'Session directory not found ({sess_dir})')
+
+    ma_filepaths = [fp.as_posix() for fp in sess_dir.glob('*.json')]
```

Comment:
Same comment as above, matching `.json` is a poor indication for `miniscope_analysis` type, perhaps something more specific